src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -306,6 +306,10 @@ export default {
 		expLabel: {
 			type: String,
 			default: ''
+		},
+		lendNowButton: {
```

Comment:
Prop to display add directly to basket button instead of view loan. It doesn't feel great to pass this prop through KivaClassicSingleCategoryCarousel.vue > KivaClassicLoanCarousel.vue > /KivaClassicBasicLoanCard.vue but the previous approach (of checking the route for the funded borrower profile is also not very good.

Reply:
It think this an appropriate prop to add as it applies to "all" loan cards in the carousel and will be very useful! I can now add it as a config on the contenful version and we'll be able to use that in other places where we want folks to add to basket!